play/PlayerRules/CharacterCreation/02_Classes.txt

```diff
@@ -2,645 +2,696 @@ Core classes:
-|
-|
-|
-V
-The Trail becomes bright against a monochrome background of edged shades of 
-grey. +75% to a tracking check, or maximum 2 turns to pass.
-----------------------
-Highlight Target: (Perception checks do not include checks to hit target. But by using a perception check for an 
-action to see where the target is and passing the target''s Cover DC+25 the player may recieve a +5% / +10% accuracy bonus.)
-Expend 10 nanites to mark a target. All damage dealt by allies to this target 
-is increased by 10, and any perception checks are 25% easier to pass. Use once 
-per encounter. Maximum of Level number uses per day. Lasts for duration of 
-encounter
-|
-|
-|
-V
-Expend 10 nanites to paint a target. All damage dealt by allies to this target 
-is increased by 20, and any perception checks are 50% easier to pass. Use 
-twice per encounter. Level number of uses per day. Lasts for the day.
-
-
-"Paladin" - You are a righteous warrior, vetted and trained by one of the 
+	----------------------
+	Hunter Vision - Lvl/2 uses per day
+	Your breathing calms, as time appears to slow down. Your vision turns into 
+	myriad shades of grey. You see a faint path of color representing the trail of 
+	your quarry. +20% to a tracking check, Or take 3 turns to pass the check. 
+	Passively, gain the ability to sense where your Marked Quarry is, within
+	10 meters of you.
+	|
+	|
+	V
+	The Trail becomes bright against a monochrome background of edged shades of 
+	grey. You can see marks and paths that others simply cannot. +50% to a tracking 
+	check, or maximum 2 turns to pass. Passively, gain the ability to sense where your
+	Marked Quarry is, within 20 meters of you.
+	----------------------
+	Highlight Quarry - 
+	If your Quarry is marked, for 1 action, you can activate a Marked Quarry's nanites to make them 
+	more visible. All damage dealt to this target is increased by 10. Two uses per encounter, one use
+	per Quarry. Maximum of Lvl number uses per day. Lasts for the duration of the encounter.
+	After level 10, all damage dealt to this target is increased by 20 (instead of 10).
+	----------------------
+	They Never Saw it Coming -
+	If your Quarry has not seen you for at least 1 turn, your next attack against them does +30 bonus damage.
+	Triggerable once per Quarry.
+	Increases to +50 bonus damage at level 10.
+======================================
+
+
+=====   PALADIN   =====
+You are a righteous warrior, vetted and trained by one of the 
 powerful corporations or some other technologically advanced entity. You seek 
-to uphold the principals which define the group you''ve sworn allegience to. 
+to uphold the principals which define the group you've sworn allegiance to. 
 Through this devotion you have been granted access to high tech in order to 
 pursue your goals and missions.
-Passives: You gain 45 total nanites and each level you gain an additional 5 
-maximum nanites for each level after your first.
+Class Primary Attribute: Fortitude
```

Comment:
I can see Fort, but I would have expected Cha, especially after moving Priests off of it.